wiki/Manual.md

```diff
@@ -2826,6 +3031,23 @@ $exchange = new $exchange_class (array (
 
 Note that your private requests will fail with an exception or error if you don't set up your API credentials before you start trading. To avoid character escaping **always write your credentials in single quotes**, not double quotes (`'VERY_GOOD'`, `"VERY_BAD"`).
 
+### Sign In
+
+Some exchanges required you to sign in prior to calling private methods, which can be done using the `signIn` method
+
+```Javascript
+signIn (params = {})
+```
+
+Parameters
```

Comment:
I would make it an `#### Arguments` subheader throughout the docs.

Reply:
Function parameters are the names listed in the function's definition. Function arguments are the real values passed to the function. Parameters are initialized to the values of the arguments supplied.

Reply:
Yes, you're right.